easybuild/easyconfigs/q/qmflows/qmflows-1.0.0-foss-2022a.eb

```
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'qmflows'
+version = '1.0.0'
+
+homepage = 'https://github.com/SCM-NV/qmflows'
+description = """This library tackles the construction and efficient execution 
+of computational chemistry workflows. This allows computational chemists to use 
+the emerging massively parallel compute environments in an easy manner and focus 
+on interpretation of scientific data rather than on tedious job submission 
+procedures and manual data processing.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = {
+    ('Python', '3.10.4'),
+    ('RDKit', '2022.09.4', '', ('foss', '2022a')),
+    ('HDF5', '1.12.2', '', ('gompi', '2022a')),
+    ('SciPy-bundle', '2022.05', '', ('foss', '2022a')),
+    ('h5py', '3.7.0', '', ('foss', '2022a')),
```

toolchain versions are not necessary here, can be removed
```suggestion
    ('RDKit', '2022.09.4'),
    ('HDF5', '1.12.2'),
    ('SciPy-bundle', '2022.05'),
    ('h5py', '3.7.0'),
```